safe/impact_function_v4/test/test_impact_function.py

```diff
@@ -89,6 +89,30 @@ def run_scenario(scenario):
 class TestImpactFunction(unittest.TestCase):
     """Test Impact Function."""
 
+    def test_keyword_monkey_patch(self):
+        """Test behaviour of generating keywords"""
+        exposure_path = standard_data_path('exposure', 'building-points.shp')
+        # noinspection PyCallingNonCallable
+        exposure_layer = QgsVectorLayer(exposure_path, 'Building', 'ogr')
```

Comment:
You should use 'safe.test.utilities.load_test_vector_layer' it will manage monkey patching for you